chart/scripts/post-delete-hook.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+set -e
+
+succeeded=()
+failed=()
+echo "Uninstalling Rancher resources in the following namespaces:" "$namespaces"
+for namespace in $namespaces; do
+  app_list=$(helm list -n "$namespace" -q)
+  for app in $app_list; do
+    if [[ $app =~ .crd$ ]]; then
+      echo "--- Skip the app [$app] in the namespace [$namespace]"
+      continue
+    fi
+    echo "--- Deleting the app [$app] in the namespace [$namespace]"
+    if [[ ! $(helm uninstall "$app" -n "$namespace") ]]; then
```

Comment:
Does the following work instead?

```suggestion
    if ! helm uninstall "${app}" -n "${namespace}"; then
```

Reply:
I'd prefer to keep the `[[ ]]` to keep the consistency of the format.